### PR TITLE
jbang: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/jbang.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+> Crea, edita y ejecuta fácilmente programas Java autocontenidos de sólo código fuente.
+> Ver también: `java`.
+> Más información: <https://www.jbang.dev/documentation/guide/latest/cli/jbang.html>.
+
+- Inicializa una clase Java simple:
+
+`jbang init {{ruta/al/archivo.java}}`
+
+- Inicializar una clase Java (útil para scripts):
```
Comment (tricantivu) [inline pages.es/common/jbang.md:11]:
```suggestion
- Inicializa una clase Java (útil para scripts):
```

**Review by tricantivu — CHANGES_REQUESTED** [review]
What do you think about using a preposition, for example:

```md
> programas **en** Java
```

**Comment by kant** [comment]
> What do you think about using a preposition @kant?
> 
> ```
> > ... programas en Java
> ```

Done

File: pages.es/common/jbang.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+`jbang run --interactive`
+
+- Configura un proyecto temporal para editar un script en un IDE:
+
+`jbang edit --open={{codium|code|eclipse|idea|netbeans|gitpod}} {{ruta/al/script.java}}`
+
+- Ejecuta un fragmento de código Java (Java 9 y posteriores):
+
+`{{echo 'Files.list(Paths.get("/etc")).forEach(System.out::println);'}}} | jbang -`
```
Comment (kbdharun) [inline pages.es/common/jbang.md:25]:
```suggestion
`{{echo 'Files.list(Paths.get("/etc")).forEach(System.out::println);'}} | jbang -`
```

**Review by kbdharun — APPROVED** [review]

**Review by tricantivu — APPROVED** [review]
The pull request is becoming too big, the other suggestions may be addressed later.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**